modules/vaos/app/services/vaos/messages_service.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require_relative '../vaos/concerns/headers'
+
+module VAOS
+  class MessagesService < Common::Client::Base
+    include Common::Client::Monitoring
+    include SentryLogging
+    include VAOS::Headers
+
+    configuration VAOS::Configuration
+
+    attr_reader :user, :request_id
+
+    STATSD_KEY_PREFIX = 'api.vaos'
+
+    def initialize(user, request_id)
+      @user = user
+      @request_id = request_id
+    end
+
+    def get_messages
+      with_monitoring do
+        response = perform(:get, messages_url, headers)
```

Comment:
Wondering if the response portion of this code could be refactored out into a response class for uniformity purposes?

Reply:
There might be an opportunity for doing a custom base class that makes this more DRY, but so that "with_monitoring" for example happens automatically, but I think I favor composability over abstraction here.

Can you point me to examples of where response was abstracted into a class in other services/clients?